core/templates/components/summary-tile/learner-story-summary-tile.component.html

```diff
@@ -32,6 +32,61 @@
     </a>
   </mat-card>
 </div>
+<div *ngIf="displayAreaIsLearnerDashboard" class="new-learner-story-summary-tile">
+  <mat-card class="oppia-story-dashboard-card" [attr.style]="isButtonHovered()">
+    <div class="oppia-title-section" [ngStyle]="{'background-color': thumbnailBgColor}">
+      <div class="oppia-story-tags">
+        <span *ngIf="learnerGroupTitle.length !== 0" class="oppia-learnergroup-title-tag">
+          <img [src]="learnerGroupImageUrl" alt="learnergroup image">
+          <span>
+            {{ learnerGroupTitle }}
+          </span>
+        </span>
+        <span *ngIf="markTileAsGoal" class="oppia-goal-tag">
+          <img [src]="todolistImageUrl" alt="todolist image" class="oppia-goal-svg-image">
+          <span>Goals</span>
+        </span>
+      </div>
+      <img class="oppia-thumbnail-image" alt="" [src]="thumbnailUrl" [attr.style]="isButtonHovered()">
+    </div>
+    <div class="oppia-summary-section">
+      <div class="next-incomplete-chapter" *ngIf="isDisplayAreaHome()">
+        {{nextIncompleteNodeTitle}}
+      </div>
+      <div class="oppia-topic-title e2e-test-topic-name-in-new-new-learner-story-summary-tile">
```

Comment:
typo: "..new-new.."

Also, here and elsewhere, please drop the "new-" part throughout the PR. Terminology is confusing, there is nothing "new" about it once merged. We're updating the look and feel of the dashboard and not maintaining 2 versions of it.

Reply:
Thanks for pointing this out, I have removed the typo and explained above why to use the new word.